packages/cli/src/commands/org/resources/pull.ts

```diff
@@ -59,6 +59,11 @@ export default class Pull extends Command {
       default: true,
       allowNo: true,
     }),
+    override: flags.boolean({
```

Comment:
I think you're misusing the word override here.
It should be overwrite I think :)

Reply:
Yep !

Reply:
Oups. thanks